kukkuu/settings.py

```diff
@@ -152,8 +154,12 @@
     AZURE_CONTAINER = env("AZURE_CONTAINER")
     if env("AZURE_BLOB_STORAGE_SAS_TOKEN"):
         SAS_TOKEN = env("AZURE_BLOB_STORAGE_SAS_TOKEN")
-        AZURE_ENDP = f"BlobEndpoint=https://{AZURE_ACCOUNT_NAME}.blob.core.windows.net"
-        AZURE_CONNECTION_STRING = f"{AZURE_ENDP};SharedAccessSignature={SAS_TOKEN};"
+        AZURE_ENDP = (
+            f"BlobEndpoint=https://{AZURE_ACCOUNT_NAME}.blob.core.windows.net"  # noqa
+        )
+        AZURE_CONNECTION_STRING = (
+            f"{AZURE_ENDP};SharedAccessSignature={SAS_TOKEN};"  # noqa
+        )
```

Comment:
I don't know what has happened here but removing the `# noqa` and running `black . && isort . && flake8` formats them as before without errors/warnings:
```python
        AZURE_ENDP = f"BlobEndpoint=https://{AZURE_ACCOUNT_NAME}.blob.core.windows.net"
        AZURE_CONNECTION_STRING = f"{AZURE_ENDP};SharedAccessSignature={SAS_TOKEN};"
```
So the `# noqa`'s are unnecessary.

Reply:
I can retry it without them now. I sure had some strange issues with them, but now I cannot remember anymore whether the issues were locally or in CI-pipelines.

I push a new commit that removes this and let's see then.

Reply:
Okay, at least locally I got this from flake8 when staged changes where linted by the precommit hook.

![image](https://github.com/City-of-Helsinki/kukkuu/assets/389204/bed813b0-430d-48e2-9801-45cb89760696)

But when I ran the flake8 manually, I don't get the same issue 🤔 It seems that my precommit runs a different version of flake8 or with different settings, or something like that.

Reply:
```
❯ flake8 --version
6.0.0 (mccabe: 0.7.0, pycodestyle: 2.10.0, pyflakes: 3.0.1) CPython 3.9.17 on Darwin
```

Reply:
I ran `black . && isort . && flake8` inside the docker container

Reply:
Does the pre-commit hook take `setup.cfg` correctly into account as it configures flake8 and isort?

Reply:
I [upgraded the flake8 from 6.0.0 to 6.1.0](https://github.com/City-of-Helsinki/kukkuu/pull/347/commits/ada35153ab5499e4fc113bed3b5535ef6e1137f2) and that fixed some issues. Also added "an improved configuration" for the pre-commit hook.

More about these:
1. https://github.com/pre-commit/pre-commit-hooks/issues/112
2. https://stackoverflow.com/a/77402153/784642.

Reply:
Now I got the same result with `pre-commit run flake8 -a -v` and `flake8 .`